tests/test_plan_epic.sh

```
@@ -0,0 +1,110 @@
+assert_file_contains "jq 文字列補間禁止の制約がある" "$SKILL_MD" "string interpolation"
+assert_file_contains "コマンド素直実行の制約がある" "$SKILL_MD" "コマンドをそのまま実行"
+assert_file_contains "介入ポイントの記述がある" "$SKILL_MD" "介入ポイント"
+
+# --- F. install.sh のプリセット分岐 ---
+
+echo "--- F. install.sh のプリセット分岐 ---"
+
+assert_file_exists "install.sh が存在する" "$INSTALL_SH"
+
+# minimal / standard プリセットで plan-epic を削除する分岐がある
+PLAN_EPIC_DELETIONS=$(grep -c 'rm -rf "\${skills_dir}/plan-epic"' "$INSTALL_SH" || echo 0)
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)